tests/phpunit/Admin/Migrations/WC_Stripe_Subscriptions_Repairer_Legacy_SEPA_Tokens_Test.php

```diff
@@ -68,7 +67,7 @@ class WC_Stripe_Subscriptions_Repairer_Legacy_SEPA_Tokens_Test extends WP_UnitTe
 	 *
 	 * @var string
 	 */
-	private $legacy_sepa_gateway_id = WC_Gateway_Stripe_Sepa::ID;
+	private $legacy_sepa_gateway_id = 'stripe_sepa';
```

Comment:
Might it be worth using a new constant for this rather than having `'stripe_sepa'` everywhere?

It also feels like it could be worth pushing just that change into a dedicated PR, as that would be easy to inspect and review, and we could get it shipped with minimal risk. That would also reduce the size of this PR by quite a few files.